src/docs/design/intro.mdx

```diff
@@ -8,18 +8,7 @@ Using the PX Blue design system as a designer can greatly improve your efficienc
 
 ## Environment
 
-We host and maintain our design assets through Figma. Figma is a powerful prototyping tool that facilitates collaboration and sharing of design assets between teams. It can be used on the web, and they also provide a desktop app for Mac and Windows. In order to use Figma, you will need to [sign up](https://www.figma.com/) for an account.
-
-## What Materials are Available?
```

Comment:
If it were me, I would keep this section here, but modify it to say something like:

"Blah blah blah, PX Blue has a number of materials available to help designers use the design system. These include our style guide (link to style guide landing page) and design patterns (link) as well as various Figma resources (link to resources page).

Reply:
I end up leaving the guideline CTAs there to make them more pronounced, since these are the items we'd like to emphasize.